contracts/ColonyFunding.sol

```diff
@@ -60,11 +60,26 @@ contract ColonyFunding is ColonyStorage, DSMath {
     return task.payouts[_role][_token];
   }
 
+  function moveFailedTaskFunds(uint256 _id, address _token) public
+  taskFinalized(_id)
+  {
+    Task storage task = tasks[_id];
+    require(task.roles[WORKER].rating <= 2);
+    uint payout = task.payouts[WORKER][_token];
+    task.payouts[WORKER][_token] = 0;
+    task.totalPayouts[_token] = sub(task.totalPayouts[_token], payout);
+
+    moveFundsBetweenPots(task.potId, 1, payout, _token);
+  }
+
   function claimPayout(uint256 _id, uint256 _role, address _token) public
   taskFinalized(_id)
   {
     Task storage task = tasks[_id];
     require(task.roles[_role].user == msg.sender);
+    if (_role == WORKER) {
```

Comment:
The above solution means we won't need this check here as well, as the worker payout is `0`.